toolkit/scripts/toolchain/build_official_toolchain_rpms.sh

```diff
@@ -585,6 +585,9 @@ chroot_and_install_rpms pyproject-rpm-macros pyproject-srpm-macros
 # ocaml and other ocmal packages require ocaml-srpm-macros 
 build_rpm_in_chroot_no_install ocaml-srpm-macros
 
+build_rpm_in_chroot_no_install python-packaging
+chroot_and_install_rpms python-packaging python3-packaging
+# rebuild python-packaging to resolve circular dependency
 build_rpm_in_chroot_no_install python-packaging
 chroot_and_install_rpms python-packaging python3-packaging
```

Comment:
Nit-pick: no need to re-install as `python-packaging` doesn't need itself to build, we're just adding more `Provides` to the package but that doesn't change its functionality.

Reply:
Removed

Reply:
Thanks, but I think now we accidentally removed a rebuild of `python-packaging`. And we'd need a `Release` bump in the spec as well.

Reply:
The re-install is there. Did it end up making a difference and we need to keep it?

Reply:
Forgot to remove this part; I don't think it's necessary. Will remove and re-run the test builds